services/github/github-issue-detail-redirect.service.js

```diff
@@ -27,7 +27,7 @@ module.exports = [
         ':which(state|title|author|label|comments|age|last-update)/:user/:repo/:number([0-9]+)',
     },
     transformPath: ({ which, user, repo, number }) =>
```

Comment:
Two items in this file:

1) this entire redirector (from lines 22-31) should be removed
2) the `trasformPath` response above on line 19 should be
` /github/${issueKind}/detail/${whichMap[which]}/${user}/${repo}/${number}`,